### ISSUE TITLE
SendDirectData returns 1

### ISSUE THREAD

**Issue by rsp4jack** [KaleidonKep99/Keppy-s-MIDI-Driver#241]
**Problem and Steps to reproduce**
I am trying to use KDMAPI (without winmm functions):
```c++
auto mod = LoadLibrary("OmniMIDI");
... (Assign functions)

cout << IsKDMAPIAvailable() << endl;
InitializeKDMAPIStream();
SendDirectData(0x8189); // This step returns 1
TerminateKMDAPIStream();
```

**Environment**
OmniMIDI version 14.6.13.0
======== System  information ========
Driver version: 14.6.13.0
Operating system: Windows 10 Home (10.0.19044.1889, AMD64)
CPU: Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz (AMD64 architecture)
CPU info: GenuineIntel, 4 cores, 4 threads, 3.40GHz (3401MHz)
GPU: Intel(R) HD Graphics 630
GPU chip: Intel(R) HD Graphics Family
GPU info: 1024MB VRAM, driver version 31.0.101.2111
Total memory: 8057MB (8.448.958.464 bytes)
Available memory: 981MB (12.2%, 1.029.603.328 bytes)
Machine type: Desktop

**Additional context**
SendDirectLongData success

**Comment by KaleidonKep99** [comment]
Hi!

SendDirectData is a void and does not return an integer.

**Comment by rsp4jack** [comment]
https://github.com/KeppySoftware/OmniMIDI/blob/master/DeveloperContent/KDMAPI.md#senddirectdata
It returns HRESULT in docs.
And C#: https://github.com/KeppySoftware/OmniMIDI/blob/master/DeveloperContent/OmniMIDI.cs

**Comment by KaleidonKep99** [comment]
I just noticed that the docs are outdated oops!
I'll update them accordingly, the function will now quietly fail.

**Comment by rsp4jack** [comment]
Fixed in ec6912a46cedbda9a1cf690847771bbb417e6dbd
